client/my-sites/plugins/plugin-sections/index.jsx

```diff
@@ -107,6 +107,13 @@ class PluginSections extends React.Component {
 					textOnly: true,
 				} ),
 			},
+			{
+				key: 'faq',
+				title: this.props.translate( 'Support', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **52** times:
`translate( 'Support', { context: 'logged-out-homepage'} )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/support-section-to-plugins)


:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `Filter label for theme content`
- `Support page for this theme`

 Would it make sense to reuse one of the above?